Discovery/Discovery/Presentation/WebDiscovery/DiscoveryURIDetails.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  DiscoveryURIDetails.swift
+//  Discovery
+//
+//  Created by SaeedBashir on 12/18/23.
+//
+
+import Foundation
+import Core
+
+// Define your uri scheme
+public enum URIString: String {
+    case appURLScheme = "edxapp"
+    case pathPlaceHolder = "{path_id}"
+}
+
+public  enum URLParameterKeys: String, RawStringExtractable {
```

Comment:
@saeedbashir, can we remove extra space here?
`public  enum` -> `public enum`

Reply:
Done